package.json

```
@@ -101,16 +99,14 @@
     "busboy": "^0.3.1",
     "celebrate": "^13.0.3",
     "compression": "~1.7.2",
-    "connect": "^3.6.6",
     "connect-mongo": "^3.2.0",
     "convict": "^6.0.0",
     "convict-format-with-validator": "^6.0.0",
     "cookie-parser": "~1.4.0",
     "crypto-js": "^4.0.0",
     "css-toggle-switch": "^4.1.0",
     "csv-string": "^4.0.1",
-    "dedent-js": "^1.0.1",
-    "deep-diff": "^1.0.1",
+    "dedent-js": "~1.0.1",
```

Just to confirm if this should not be exact version instead, based on the reasoning that we are using a package last published 5 years ago and don't want to allow any updates at all without reviewing closely

```suggestion
    "dedent-js": "1.0.1",
```

---

my logic is that if someone decides they want to patch any bugs in the library, we should be able to apply it. wdyt?

---

sounds good!